setup/templates/steps/compose/01_config.html

```diff
@@ -0,0 +1,67 @@
+{% call macros.panel("info", "Step 2 - Initial configuration") %}
+<p>Before starting some variables must be set</p>
+
+<div class="form-group">
+  <label>Root path: </label>
+<!--   Validates path -->
+  <input class="form-control" type="text" name="root" value="/mailu" required pattern="^/[-_A-Za-z0-9]+(/[-_A-Za-z0-9]*)*">
+</div>
+
+<div class="form-group">
+  <label>Main mail domain</label>
+<!--   Validates domain name -->
+  <input class="form-control" type="text" name="domain" placeholder="e.g. mailu.io" 
+  		required pattern="^([a-z0-9]+(-[a-z0-9]+)*\.)+[a-z]{2,}$">
+</div>
+
+<div class="form-check form-check-inline">
+  <label class="form-check-label">
+    <input class="form-check-input" type="checkbox" name="domain_registration" value="true">
+    Domain registration
+  </label>
+</div>
+
+<div class="form-group">
+  <label>Postmaster local part (will append the main mail domain)</label>
+  <input class="form-control" type="text" name="postmaster" value="admin" required>
+</div>
+
+<div class="form-group">
+  <label>Choose how secure connections will behave </label>
```

Comment:
Definitely not agreeing with the label. Except for ``notls`` (which we ought to have a discussion about), the security level does not really change when changing this. Maybe "Choose how you wish to handle security (TLS) certificates" and add a link to the documentation for more details.